src/chapters/sessions/managing-cookies.rst

```diff
@@ -0,0 +1,183 @@
+Flask Cookies
+=============
+
+When a server executes code to create a cookie, part of the command sets a time
+limit for how long to save the data. If no time is given in the code, then the
+cookie gets erased when we quit the browser. Putting our device to sleep or
+closing the tab does NOT clear cookies!
+
+Recall that Flask runs a server on our machine when we launch an application.
+This means we can include some Python code to set and access our own cookies.
+
+Let's see how to do this.
+
+Cookie Video
+------------
+
+Watch this short clip to learn how to set cookies with Flask. You can find the
+starter code in this `GitHub repository <https://github.com/LaunchCodeEducation/LCHS_cookie_intro.git>`__.
+By switching between the ``main`` to ``final-code`` branch, you can view both
+the starting and ending points of the demonstration.
+
```

Comment:
I had to create the env and install flask, maybe a  link to 21.2?  or like 23.5.1 #5

Reply:
I added an admonition box to clarify.  Let me know if it works for you.

Reply:
That works!  I finally just copied the information to my own cheatsheet.